File: src/iOS/RotatingNavigationController.m
//
//  RotatingNavigationController.m
//  Go Map!!
//
//  Created by Bryce on 2/21/13.
//  Copyright (c) 2013 Bryce. All rights reserved.
//

#import "RotatingNavigationController.h"

@implementation RotatingNavigationController


- (BOOL)shouldAutorotateToInterfaceOrientation:(UIInterfaceOrientation)interfaceOrientation
{
    return YES;
}

- (BOOL)shouldAutorotate
{
	return YES;
}

- (NSUInteger)supportedInterfaceOrientations
{
	return UIInterfaceOrientationMaskAll;
}

@end

File: src/iOS/RotatingNavigationController.swift
//
//  RotatingNavigationController.swift
//  Go Map!!
//
//  Copyright © 2021 Bryce Cogswell. All rights reserved.
//

import UIKit

class RotatingNavigationController: UINavigationController {
	override var shouldAutorotate: Bool {
		return true
	}

	override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
		return .all
	}
}
